.github/workflows/tmate_debug/action.yml

```diff
@@ -18,3 +18,5 @@ runs:
     - name: set up tmate session for debugging
       if: env.USE_TMATE
       uses: mxschmitt/action-tmate@v3
+      with:
```

Comment:
is this needed?

Reply:
yes, we need it false, and doing one place using composite is the best place